manticore/core/executor.py

```diff
@@ -487,9 +487,20 @@ def generate_testcase(self, state, message = 'Testcase generated'):
         assert solver.check(state.constraints)
         for symbol in state.input_symbols:
             buf = solver.get_value(state.constraints, symbol)
-            file(self._getFilename('test_%08x.txt'%test_number),'a').write("%s: %s\n"%(symbol.name, repr(buf)))
+            open(self._getFilename('test_{:08x}.txt'.format(test_number)),'a').write("{:s}: {:s}\n".format(symbol.name, repr(buf)))
         
-        file(self._getFilename('test_%08x.syscalls'%test_number),'a').write(repr(state.platform.syscall_trace))
+        open(self._getFilename('test_{:08x}.syscalls'.format(test_number)),'a').write(repr(state.platform.syscall_trace))
+
+        # save symbolic files
+        files = getattr(state.platform, 'files', None)
```

Comment:
after some discussion in slack, given the impending huge refactor in #241 it should be fine to leave this now and refactor later. my proposal is to implement some method on the Platform for, generating platform specific inputs/data/results. for example `platform.generate_misc_data()` which returns a dict of filename/name of data -> data (str)